worker/introspection/script.go

```diff
@@ -114,7 +114,9 @@ juju_statetracker_report () {
 }
 
 juju_machine_lock () {
-  juju_agent machinelock 2> /dev/null
+  for agent in $(ls /var/lib/juju/agents); do
```

Comment:
Are these sorted by default? QA steps seem to say yes?

Reply:
`ls` does sort by default.